src/app.rb

```diff
@@ -41,16 +39,14 @@ def show_interactive_console
     puts 'Please choose an option by entering a number:'
     puts '1. List all books'
     puts '2. List all music albums'
-    puts '3. List all movies'
-    puts '4. List all games'
-    puts '5. List all genres'
-    puts '6. List all labels'
-    puts '7. List all authors'
-    puts '8. List all sources'
-    puts '9. Add a book'
-    puts '10. Add a music album'
-    puts '11. Add a movie'
-    puts '12. Add a game'
+    puts '3. List all games'
+    puts '4. List all genres'
+    puts '5. List all labels'
+    puts '6. List all authors'
+    puts '7. Add a book'
+    puts '8. Add a music album'
+    puts '9. Add a movie'
```

Comment:
here as well

Reply:
Thanks for highlighting! I will resolve it.

Reply:
Thanks for the observation, some are meant to be for a fourth person which our team does not have.